save-backend/src/main/kotlin/com/saveourtool/save/backend/controllers/ExecutionController.kt

```diff
@@ -190,7 +191,7 @@ class ExecutionController(private val executionService: ExecutionService,
         Permission.DELETE,
         messageIfNotFound = "Could not find the project with name: $name and owner: $organizationName or related objects",
     )
-        .map { project ->
```

Comment:
blockingMap prohibits different context, because until first one context is not over, the second one wont be started for one execution?

Reply:
nope, blockingMap is useful here because `executionService.deleteExecutionExceptParticipatingInContestsByProjectNameAndProjectOrganization(project.name, project.organization)` is blocking operation (database update inside transaction). It does use a special scheduler which cannot be suspended till the operation is ended.
The main fix is in updating test data and adding constrain on database level

Reply:
> The main fix is in updating test data and adding constrain on database level

but this does not exclude the case, when we will add several contests for one execution, isnt it?
now we will just fail on such case, but the main algorithm allow such behavior